agent/agentbootstrap/bootstrap.go

```diff
@@ -106,7 +107,7 @@ func InitializeState(
 	info.Tag = nil
 	info.Password = c.OldPassword()
 
-	if err := database.BootstrapDqlite(stdcontext.TODO(), database.NewNodeManager(c, logger), logger); err != nil {
+	if err := database.BootstrapDqlite(stdcontext.TODO(), database.NewNodeManager(c, logger, databaselogger.NoopSlowQueryLogger{}), logger); err != nil {
```

Comment:
`BootstrapDqlite` invokes `WithTracingOption`, but we always pass a no-op logger.

Reply:
Yeah, good spot. The `WithTracingOption` can be removed now.